web/plugins-head.html

```diff
@@ -24,6 +24,13 @@ <h3 id="linux-install">GNU/Linux</h2>
 
 <p>In the portable builds, it&#8217;s possible to unzip directly into deadbeef folder.</p>
 
+<h3 id="windows-install">Windows</h2>
+
+<p>Download and extract .zip into %APPDATA%/deadbeef/</p>
```

Comment:
This won't be understood by most windows users.
Could this be changed to something more user friendly?

Reply:
hmm, the idea is that users copy and paste that location into File Explorer. I could add "(copy and paste the location in File Explorer)", but in my opinion it is the first thing users will do. To make it more clear I could put every path in a `<code>` block. With current css it would look like this (on firefox): 
![image](https://user-images.githubusercontent.com/6359901/105249079-cdfbd000-5b77-11eb-8b62-92115b1d8e56.png)

You might notice that `<code>` blocks are not centered to the text, as text has margin bottom of 10px, I don't know how to fix it without changing the look of whole website.

Reply:
Ok, this is a good point. I agree, please do it this way.